pyproject.toml

```diff
@@ -49,7 +49,7 @@ dev = [
 
 # https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#dynamic-metadata
 [tool.setuptools.dynamic]
-readme = {file = ["README.md", ]}
+readme = {file = "README.md", content-type = "text/markdown"}
```

Comment:
Having a *.md extension _should_ mean this is required to be rendered as markdown, but it wasn't for some reason. This way I forced the content-type so pypi would render it properly.